src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+@import "../utils/variables";
```

Comment:
You've imported the variables 3 times, but you can do it once in the `main.scss` file.

Fix this!